Intersect (Core)/Config/NpcOptions.cs

```diff
@@ -1,39 +1,151 @@
-        public bool AllowResetRadius = false;
+		/// <summary>
+		/// Configures the radius in which an NPC is allowed to move after starting to fight another entity.
+		/// </summary>
+		public int ResetRadius
+		{
+			get;
+			set;
+		} = 8;
 
-        /// <summary>
-        /// Configures the radius in which an NPC is allowed to move after starting to fight another entity.
-        /// </summary>
-        public int ResetRadius = 8;
+		/// <summary>
+		/// Configures whether or not the NPC is allowed to gain a new reset center point while it is still busy moving to its original reset point.
+		/// NOTE: Can be used to allow the NPCs to be dragged far far away, as it constantly resets the center of its radius!!!
+		/// </summary>
+		public bool AllowNewResetLocationBeforeFinish
+		{
+			get;
+			set;
+		} = false;
 
-        /// <summary>
-        /// Configures whether or not the NPC is allowed to gain a new reset center point while it is still busy moving to its original reset point.
-        /// NOTE: Can be used to allow the NPCs to be dragged far far away, as it constantly resets the center of its radius!!!
-        /// </summary>
+		/// <summary>
+		/// Configures whether or not the NPC should completely restore its vitals and statusses once it resets.
+		/// </summary>
+		public bool ResetVitalsAndStatusses
+		{
+			get;
+			set;
+		} = false;
 
-        public bool AllowNewResetLocationBeforeFinish = false;
+		/// <summary>
+		/// Configures whether or not the level of an Npc is shown next to their name.
+		/// </summary>
+		public bool ShowLevelByName
+		{
+			get;
+			set;
+		} = false;
 
-        /// <summary>
-        /// Configures whether or not the NPC should completely restore its vitals and statusses once it resets.
-        /// </summary>
-        public bool ResetVitalsAndStatusses = false;
+		/// <summary>
+		/// Configures whether or not to display Npc's Tags.
+		/// NOTE: Npc Tag Sprites are always loaded from the "tags" resource folder.
+		/// *Recommended sizes: 32x16, 64x32, 128x64 and so on [2:1 px]
+		/// </summary>
+		public bool ShowTags
+		{
+			get;
+			set;
+		} = false;
 
-        /// <summary>
-        /// Configures whether or not the level of an Npc is shown next to their name.
-        /// </summary>
-        public bool ShowLevelByName = false;
+		/// <summary>
+		/// Configures whether or not to ONLY allow Custom Tags for Npcs.
+		/// NOTE: default tags won't be rendered on NPCs anymore.
+		/// </summary>
+		public bool ShowCustomTagsOnly
+		{
+			get;
+			set;
+		} = false;
 
-    }
+		/// <summary>
+		/// Configures the position of the Tags. Only works if ShowTags = true.
+		/// 0: Positions the tag 2 pixels above the name label.
+		/// 1: Positions the tag 2 pixels under the name label.
+		/// 2: Positions the tag as prefix (2 pixels left from the name label).
+		/// 3: Positions the tag as suffix (2 pixels right from the name label).
+		/// </summary>
+		public Enums.NpcTagPos TagPosition
```

Comment:
Add a `using Intersect.Enums;` line at the top instead of `Enums.<type name>`

Reply:
👍🏽 

```
using Intersect.Enums;

namespace Intersect.Config {

		/// <summary>
		/// Configures the position of the Tags. Only works if ShowTags = true.
		/// Above: Positions the tag 2 pixels above the name label.
		/// Under: Positions the tag 2 pixels under the name label.
		/// Prefix: Positions the tag as prefix (2 pixels left from the name label).
		/// Suffix: Positions the tag as suffix (2 pixels right from the name label).
		/// </summary>
		public TagPosition TagPosition {
			get;
			set;
		} = TagPosition.Above;
```